app/services/dependency-resolver.js

```diff
@@ -2,31 +2,36 @@
 import Ember from 'ember';
 import config from '../config/environment';
 import { task, timeout } from 'ember-concurrency';
+import compareVersions from 'compare-versions';
 
-const { computed, inject, RSVP, testing } = Ember;
+const { computed, deprecate, inject, RSVP, testing } = Ember;
 
-const EMBER_VERSIONS = ['2.18.2', '2.17.2', '2.16.2', '2.15.3', '2.14.1', '2.13.0', '2.12.0', '2.11.2', '2.10.2', '2.9.1', '2.8.2', '2.7.3', '2.6.2', '2.5.1', '2.4.5', '2.3.2', '2.2.2', '2.1.2', '2.0.3', '1.13.13', '1.12.2'];
-const EMBER_DATA_VERSIONS = ['2.18.2', '2.17.1', '2.16.3', '2.15.3', '2.14.10', '2.13.2', '2.12.2', '2.11.3', '2.10.0', '2.9.0', '2.8.1', '2.7.0', '2.6.2', '2.5.5', '2.4.3', '2.3.3', '2.2.1', '2.1.0', '2.0.1', '1.13.15'];
+const EMBER_VERSIONS = ['3.2.0', '3.1.2', '3.0.0', '2.18.2', '2.17.2', '2.16.2', '2.15.3', '2.14.1', '2.13.0', '2.12.0'];
+const EMBER_DATA_VERSIONS = ['3.1.1', '3.0.2', '2.18.2', '2.17.1', '2.16.4', '2.15.3', '2.14.10', '2.13.2', '2.12.2'];
 
 const VERSION_REGEX = /^\d+.\d+.\d+(-beta\.\d+)?$/;
 
 const CDN_MAP = {
   'ember': {
+    pakage: 'ember-source',
     library: 'ember.js',
     fileName: 'ember.debug.js'
   },
 
   'ember-template-compiler': {
+    pakage: 'ember-source',
```

Comment:
typo here, should be `package`

Reply:
haha, package turned out to be a reserved word. So I misspelled it.

Reply:
ha! kk